src/Array/Extra.elm

```diff
@@ -457,25 +442,24 @@ resizelRepeat lengthNew padding =
     --> empty
 
 -}
-resizerRepeat : Int -> element -> (Array element -> Array element)
-resizerRepeat lengthNew defaultValue =
-    \array ->
-        let
-            arrayLength =
-                array |> length
-        in
-        case compare arrayLength lengthNew of
-            GT ->
-                array |> slice (arrayLength - lengthNew) arrayLength
-
-            LT ->
-                append
-                    (repeat (lengthNew - arrayLength) defaultValue)
-                    array
-
-            EQ ->
+resizerRepeat : Int -> a -> Array a -> Array a
+resizerRepeat lengthNew defaultValue array =
+    let
+        arrayLength =
+            array |> length
+    in
+    case compare arrayLength lengthNew of
+        GT ->
+            array |> slice (arrayLength - lengthNew) arrayLength
+
+        LT ->
```

Comment:
These are basically `padRight` and `padLeft`, no? I think they are useful because there are algorithms that need a padded array to work, but the names are terrible. Should be marked for deprecation and renamed in the next release

Reply:
Also we're not going to rename API functions untill we come to a real resolution on #5. So I'll resolve those comments for now.